apps/drec-ui/src/apps/certificate/logic/redemptions/redemptionsReport.ts

```diff
@@ -25,19 +25,19 @@ const formatRedemptionsReportData: TFormatRedemptionsReportData = ({
               );
 
               return {
-                  id: `${certificate.id};${certificate.claimData.periodStartDate}`,
-                  fuelCode: getFuelNameFromCode(deviceGroup.fuelCode, allFuelTypes),
-                  country: deviceGroup.countryCode,
-                  capacityRange: deviceGroup.capacityRange,
-                  installations: deviceGroup.installationConfigurations
+                  id: `${certificate.id};${index}`,
```

Comment:
This is bad, but no other options for now, because all of the certificate properties can overlap - causing errors